libs/elements/src/side-navigation/vl-side-navigation.element.ts

```diff
@@ -31,10 +36,28 @@ export class VlSideNavigation extends BaseElementOfType(HTMLElement) {
     }
 
     _rerender() {
-        setTimeout(() => {
+        const sideNavigationId = this.getAttribute('side-navigation-id');
+
+        /* 
+            Hack voor UIG-2897
+            Proza messages werden niet afgebeeld tot er een resize van de window gebeurde.
+            Door de undress() - dress() niet in een timeout te wrappen is dit probleem opgelost.
+            Dit zorgt echter voor andere problemen als de side navigation overgaat van desktop naar mobile view.
+            Omdat dit component bij elke resize volledig afgebroken en opnieuw opgebouwd wordt, is het onmogelijk om state bij te houden als er geresized wordt.
+            Als oplossing moet de afnemer een uniek side-navigation-id attribuut meegeven, dit id stoppen we in een static variabele. 
+            Als de static variabele nog niet is ingevuld of is ingevuld met het id van een andere side-navigation,
+            kunnen we er van uitgaan dat dit de eerste keer is dat deze side-navigation rendert en de correcte code uitvoeren.
+        */
+        if (sideNavigationId && VlSideNavigation.initializedSideNavigationId !== sideNavigationId) {
+            VlSideNavigation.initializedSideNavigationId = sideNavigationId;
             this._undress();
```

Comment:
Mogelijks kan `this._undress();` hier weggelaten worden, gezien het de eerste keer is dat de side-nav zichzelf opbouwt, maar het zal niets breken om dit te laten staan.

Reply:
Ik ga voor de zekerheid de this._undress() laten staan, ik heb het gevoel dat we anders problemen zouden kunnen hebben.

Reply:
Sounds good. I approve!